stix_shifter_modules/msatp/stix_translation/json/from_stix_map.json

```diff
@@ -53,8 +53,7 @@
         "DeviceInfo.DeviceName",
         "DeviceAlertEvents.DeviceName",
         "DeviceImageLoadEvents.DeviceName",
-        "DeviceLogonEvents.DeviceName",
```

Comment:
DeviceNetworkInfo - needs to be re-added
DeviceLogonInfo - needs to be deleted
DeviceImageLoadEvents.DeviceName - needs to be deleted